### PR TITLE
Update week_two.md

### PR DESCRIPTION
**Pull request by AliSchlereth**
Added some review questions and the self assessment section for your review.
@s-espinosa

### CODE REVIEW

File: module-3/backend/week_two.md [hunk 1]
```diff
@@ -4,9 +4,28 @@ Fork this respository. Answer the questions to the best of your ability. Try to
...
+8. How do you create the following table in SQL? In Active Record?   
+   (Users table with columns first_name, last_name, email, and age)   
+9. Writing Files: given a text file located at "~/Documents/pizza.txt", write code to read the file from the filesystem, then    write a new file at "~/Documents/line_count.txt" containing the number of lines in the original file.  
+
+#### Self Assessment  
+Rate yourself on the following scale.
+4. I know and understand all these concepts and did not have to look anything up  
+3. I know and understand most of these concepts but had to look something up  
+2. I am uncertain about some of these concepts and had to look some things up *  
+1. I am feeling lost about with these concepts and had to look many things up *
```
Comment (AliSchlereth) [inline module-3/backend/week_two.md]: This formatting comes out kind of wonky.  Will you double check my MD?   1. gets indented

File: module-3/backend/week_two.md [hunk 2]
```diff
@@ -4,9 +4,28 @@ Fork this respository. Answer the questions to the best of your ability. Try to
...
+4. Why do we want to create services?
+5. Why is it good practice to create PORO's with the data received from an API?
+6. What do we use VCR for? Why is it a good idea to use this tool?  
+
+#### Review  
+
+7. What does HTTP stand for?  
+8. How do you create the following table in SQL? In Active Record?   
+   (Users table with columns first_name, last_name, email, and age)   
+9. Writing Files: given a text file located at "~/Documents/pizza.txt", write code to read the file from the filesystem, then    write a new file at "~/Documents/line_count.txt" containing the number of lines in the original file.
```
Comment (AliSchlereth) [inline module-3/backend/week_two.md]: Would you rather this be another enumerable question? Perhaps this group_by question?
Given the the array (alphabetical = ["aardvark", "art", "airplane", "boy", "burp", "boot", "green", "goop", "super"]), create a Hash where the keys are the first letter of words, and the values are the list of words that share that first letter.

Comment (s-espinosa) [inline module-3/backend/week_two.md]: @AliSchlereth I like the file IO! Let's aim to save the other question for week 3.

File: module-3/backend/week_two.md (hunk 1 above)
Comment (s-espinosa) [inline module-3/backend/week_two.md]: I can't replicate on my machine with the markdown parser I use. My thought is we can put a leading `*` on each one and it won't try to format that 1 like a list. Try that, merge, and we can make another commit if it doesn't work out?

**Pull request merged**